docs/team/elvern18.md

```diff
@@ -10,17 +10,41 @@ assistants in the modules they are taking.
+
+* **Enhancements to existing features**: DeleteMultipleIndexCommand
+  * What it does: Allow users to delete contacts by listing multiple indexes.
+  * Justification: This new command would help users delete contacts more quickly and conveniently, helping them save time, since users no longer have to delete using a single index multiple times.
+  * Highlights: Have to refactor the original DeleteCommand into a parent class, and create another class DeleteSingleIndexCommand. It was tricky on how to accept valid indexes, and many scenarios have to be carefully thought. 
+
+* **Enhancements to existing features**: DeleteByNameCommand
+    * What it does: Allow users to delete contacts by their name.
+    * Justification: This new command would help users delete contacts more quickly and conveniently, helping them save time, since users no longer have to slowly search for the index that the person is in. This is especially time-consuming when there are a large number of contacts in the list.
+    * Highlights: Apart from having to refactor DeleteCommand, I had to think about how to handle the validity of the parsing of names, as it is no longer integers or indexes that I had to handle.
+
+* **Enhancements to existing features**: EditByNameCommand
+    * What it does: Allow users to edit contacts by their name.
+    * Justification: This new command would help users edit contacts more quickly and conveniently, helping them save time, since users no longer have to delete using a single index multiple times.
+    * Highlights: Have to refactor the original EditCommand into a parent class, and create another class EditByIndexCommand. It was tricky on how to handle the validity of the parsing of names, as it is no longer integers or indexes that I had to handle.
 
-* **Enhancements to existing features**:
-    * To be added soon
 
 * **Documentation**:
     * User Guide:
-        * To be added soon
+        * DeleteSingleIndexCommand
+        * DeleteMultipleIndexCommand
+        * DeleteByNameCommand
+        * EditByIndexCommand
+        * EditByNameCommand
+
     * Developer Guide:
-        * To be added soon
+        * DeleteSingleIndexCommand's use case
+        * DeleteMultipleIndexCommand's use case
+        * DeleteByNameCommand's use case
+        * EditByIndexCommand's use case
+        * EditByNameCommand's use case
+        * User stories
+        * Non-Functionality Requirements
+        * Glossary
 
 * **Project management**:
```

Comment:
I think you can remove contents for `Project Management`, `Community` and `Tools` if you have no plans to add them. The rest looks good to me!

Reply:
Removed the contents and updated PPP.